### ISSUE TITLE
Remove deprecation warnings of userstamp

### ISSUE THREAD

**Issue by allenwq** [Coursemology/coursemology2#265]
I get quite a few screens of these warnings when running tests, which are extremely annoying:
```
DEPRECATION WARNING: `serialized_attributes` is deprecated without replacement, and will be removed in Rails 5.0. (called from set_updater_attribute at gems/userstamp-4cb83294ff0c/lib/stampable.rb:159)
```

**Comment by lowjoel** [comment]
File a PR with the gem? :P

**Comment by lowjoel** [comment]
Fixed in #418.
